pipelines/rj_crm__webhook_wetalkie/flow.py

```
@@ -0,0 +1,47 @@
+from utils import send_discord_webhook_message, download_data_from_bigquery
+from prefect import flow
+import os
+
+
+@flow(log_prints=True)
+def rj_crm__webhook_wetalkie(
+    query: str,
+    test_mode: bool = False,
+):
+
+    rename_flow_run = rename_current_flow_run_task(new_name=f"rj_crm__webhook_wetalkie")
```

Ruff (0.14.8)

Local variable `rename_flow_run` is assigned to but never used

Remove assignment to unused variable `rename_flow_run`

(F841)